docs/source/hardware/lowpriorityFAQ.rst

```diff
@@ -0,0 +1,259 @@
+wall time ends if the private partition’s group member submits a job and requests for resources. 
+
+**2. What do I need to know about preemption and are there any drawbacks for my research group?**
+
+**For private partition owners:** 
+When low priority jobs are running on your hardware, you may experience a slight delay (~30s) when submitting your 
+first job to your private partition. This is the time it will take the scheduler to kill/preempt low priority jobs 
+running on your partition. This new service is designed to minimize that delay.
+
+**For users of the** ``lowpriority`` **partition:** 
+Submitting jobs to this partition will make your jobs preemptable. That means they may be stopped/suspended and 
+re-queued at any time. We recommend that you use strategies such as 
+`checkpointing <https://rc-docs.northeastern.edu/en/latest/best-practices/checkpointing.html>`_ to take full advantage 
+of the ``lowpriority`` partition. RC is working on `detailed documentation <https://northeastern-university-rc-public-documentation--19.com.readthedocs.build/en/19/hardware/lowpriority.html>`_
+and training sessions that will be available later in the Fall 2022 semester, once the service is ready to be released.
+
+**3. How does re-queuing of preempted (stopped/suspended) low priority jobs work?**
+
+Preempted jobs are put back in the default partition queue (``short``) and scheduled normally.
+
+**4. I already have access to one or more of the large, long, and/or multigpu partition(s), do I benefit from 
+using the ``lowpriority`` partition?**
+
+The goal of ``lowpriority`` partition is to double the resources available to Discovery users. Hence, specifying 
+the ``lowpriority`` partition in your SLURM job header gives your job a higher chance of being allocated resources, 
+even when your first choice of partition is unavailable. For e.g., when you specify 
+``#SBATCH --partition=short,lowpriority``, your job can start running on the ``lowpriority`` partition even when ``short`` 
+is unavailable. 
+
+Caveats
+===================
+
+**1. I am an owner of a private partition, will this affect my group’s job wait time?**
+
+As a PI who purchased your own equipment, jobs submitted by the members of your private partition to your own 
+resources always have the highest priority. When submitting jobs to your private partition, any low priority jobs 
+currently running on your servers will be killed and sent back to the queue, introducing a slight delay (~30s) on 
+the first job submitted. This new service is designed to favor partition owners heavily.
+
+**2. What is the downside of submitting a job to the** ``lowpriority`` **partition?**
+
+If a job is submitted to the ``lowpriority`` partition and a high priority job comes through that requires resources 
+currently occupied by the low priority job, then that low priority job will be stopped/suspended within 30s(?) and 
```

Comment:
question mark left here on purpose?

Reply:
Yes I think there is a confusion as to whether this is 15 or 30s. @gshomo could you confirm?

Reply:
@ShobanaSekar I think we can probably just leave it 30s just in case? and if it's 15s it's even better but I think most users won't mind this small difference.

Reply:
@ShobanaSekar , I agree with @marianalevineu , that better to leave it as 30s